flight/Modules/Actuator/actuator.c

```diff
@@ -61,6 +61,7 @@
 #define TASK_PRIORITY PIOS_THREAD_PRIO_HIGHEST
 #define FAILSAFE_TIMEOUT_MS 100
 #define MAX_MIX_ACTUATORS ACTUATORCOMMAND_CHANNEL_NUMELEM
+#define MULTIROTOR_MIXER_UPPER_BOUND 128
```

Comment:
It turns out that 7 bits of resolution is too little for many things. This lays the groundwork for increasing resolution without having to change too much code. I plan to open up a PR in the future where we can discuss what's the best balance of memory vs. resolution.

The type promotion was included by accident. I use the extra resolution a lot, but it's not necessary for this PR so I'm removing it.

FWIW, 128 is the original value. Changing it to 127 would probably not break anything, because it's such a minor difference.